coa/pkg/apis/v1alpha2/host/host.go

```diff
@@ -55,6 +61,7 @@ type APIHost struct {
 	Vendors              []VendorSpec
 	Bindings             []bindings.IBinding
 	SharedPubSubProvider pv.IProvider
+	shutdownGracePeriod  time.Duration
```

Comment:
Hello haishi, only hostconfig is using shotdowngraceperiod. It's using string since it's a config json file. Using time.Duration would need to implement an override json marshal and unmarshal.